src/exodus/test/sigmadb_tests.cpp

```diff
@@ -13,18 +13,45 @@
 
 #define TEST_MAX_COINS_PER_GROUP 30
 
+#define GROUPSIZE_KEY_SIZE 1
+
+// proxy
+class CMPMintList : public exodus::CMPMintList
+{
+public:
+    CMPMintList(const boost::filesystem::path& path, bool fWipe, uint16_t groupSize = 0)
+        : exodus::CMPMintList(path, fWipe, groupSize)
+    {
+    }
+
+    uint16_t GetGroupSize()
+    {
+        return exodus::CMPMintList::GetGroupSize();
+    }
+
+    uint16_t InitGroupSize(uint16_t groupSize)
+    {
+        return exodus::CMPMintList::InitGroupSize(groupSize);
+    }
+};
+
 struct DBTestSetup : TestingSetup
 {
     DBTestSetup() : TestingSetup(CBaseChainParams::REGTEST)
     {
-        p_mintlistdb_test = new CMPMintList(pathTemp / "MP_txlist_test", false, TEST_MAX_COINS_PER_GROUP);
+        p_mintlistdb_test = new exodus::CMPMintList(pathTemp / "MP_txlist_test", false, TEST_MAX_COINS_PER_GROUP);
     }
     ~DBTestSetup()
     {
         p_mintlistdb_test->Clear();
     }
 
-    CMPMintList *p_mintlistdb_test;
+    CMPMintList *GetCMPMintList(const std::string& fileName, uint16_t groupSize)
```

Comment:
Method name is quite confuse due to it create a new instant. Better to rename it to `CreateCMPMintList` instead. Also will be better to return it as `std::unique_ptr` instead.

Reply:
good idea